docs/guide/content/user_guide/api_keys.md

```diff
@@ -0,0 +1,49 @@
+# API keys
+
+Some repository features in OnDemand Loop are only available to authenticated users. API keys let the application request restricted content — such as draft datasets or unpublished depositions — from services like Dataverse and Zenodo on your behalf.
+
+Without a repository API key, OnDemand Loop can browse and download only the public versions of datasets that have already been published. Drafts or private uploads remain invisible, and attempts to load them from the interface will fail.
+
+---
+
+### Getting your Dataverse API token
+
+1. Sign in to your Dataverse installation.
+2. Click your user name in the top-right corner and select **API Token** from the menu.
+3. If you do not already have a token, click **Create Token**. Otherwise, copy the existing token value.
+4. If you already have a token, check the expiration date. If it has expired, click **Recreate Token** to obtain a new one.
+5. Keep the token secure — treat it like a password. You can regenerate or delete it at any time from the same page.
+
+!!! tip
+
+    API keys will provide access to all datasets linked to that user account, including your own datasets and those where 
+    you have been added with a role by another user. 
```

Comment:
I would clarify this or remove. I recall that there were errors when data was shared between accounts, not sure what are the permissions needed. Have you tested? If so put only the testes cases. If not, I would remove.

"API keys will provide access to all datasets linked to that user account, including your own datasets" this could mean that your are using someone else account and it will give you access to you own too

Reply:
I removed the section at the moment because I need to do more tests to ensure what roles apply to this rule